CHANGELOG.md

```diff
@@ -14,6 +14,14 @@ Releases
 This high level changelog is usually updated when a release is tagged.
 On the master branch there may be changes that are not (yet) described here.
 
+### 1.23.0
+
+* [-] Support older mac0S versions that lack vdprintf()
+* [-] Patch certificate login error for FortiOS 7.4.4
+* [-] Fix size and initialisation for array `data`
```

Comment:
Actually, I suggest you remove this line, because this PR reworks the code from the previous PR.
```suggestion
```